branchinfo.txt

```diff
@@ -4,5 +4,5 @@
 MAJOR_VERSION=1
 MINOR_VERSION=0
 PATCH_VERSION=0
-RELEASE_SUFFIX=preview1
-CHANNEL=beta
+RELEASE_SUFFIX=preview2
+CHANNEL=preview
```

Comment:
@Sridhar-MS are we able to use the existing scripts to download from the preview channel? If not, we need to update the scripts as part of this change.